src/ds/ts/add-mod-del.md

```diff
@@ -0,0 +1,66 @@
+## Manage data points
```

Comment:
I'm not sure we should introduce a new term - 'data points'. We usually call them 'samples'.